rootfs/opt/entry-point.sh

```
@@ -3,10 +3,22 @@ set -eu
 
 EMPTY=""
 
+AUTH_TYPE=${AUTH_TYPE:-"md5"}
+
 if [ ! -f $USERLIST_PATH -a -n ${DRUPAL_DB_USER:-EMPTY} -a -n ${DRUPAL_DB_PASSWORD:-EMPTY} ] ; then
```

Shellcheck (0.11.0)

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)